### ISSUE TITLE
MecanumDrive cartesian is inverted in the horizontal direction

### ISSUE THREAD

**Issue by rothso** [strongback/strongback-java#114]
This was following the recent update to MecanumDrive. It appears that left/right are now inverted for relative cartesian drive. Driving at an angle also moves the bot in the wrong directions, which leads me to think something doesn't check out in the math. Everything was working fine before the update.

**Comment by agausmann** [comment]
Yes, you are correct; the input axes were changed for 2017.2.0 so they would be consistent with the inputs of the `TankDrive#arcade()` methods. Read the latest JavaDoc and compare its expected inputs to your current input scheme to make sure this is actually a bug.
